openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
         #got sick of doing this everywhere
         return random_json["firstName"] + " " + random_json["lastName"]
 
+    def process_votes(self,rollcalls,bill,proxy):
+
+        result_types = {
+            'FAILED': False,
+            'DEFEATED': False,
+            'PREVAILED': True,
+            'PASSED': True,
+            'SUSTAINED': True,
+            'NOT SECONDED': False,
+            'OVERRIDDEN': True,
+            'ADOPTED': True,
+        }
+
+        for r in rollcalls:
+            proxy_link = proxy["url"] + r["link"]
+            (path, resp) = self.urlretrieve(proxy_link)
+            text = convert_pdf(path, 'text')
+            lines = text.split("\n")
+            os.remove(path)
```

Comment:
In general agreed. I'm going to leave it for now, though and PR it later after I've done some research on how much (ie line splitting or not) we typically want to include so we can get IN done and out.